### PR TITLE
Enh/983 stream next prev btns

### LINKED ISSUE #983: Accessibility Feature: Adding 'next' and 'previous' to offline stream page.
**Is your feature request related to a problem? Please describe.**
Accessibility Feature: When user finish watching the lecture, he cant go to the next one directly. There are no 'next' or 'previous' buttons. Instead the user needs to go back and chose the next one. 

**Describe the solution you'd like**
Adding 'next' and 'previous' would make the navigation easier.

**Describe alternatives you've considered**
Adding the list of lectures on the offline stream page would be a plus.

**Additional context**

### PR DESCRIPTION
**Pull request by mono424**
### Motivation and Context
It is unnecessarily hard to navigate to the next stream from the watch page.

### Description
It was requested to add previous and next buttons to the stream page, with optional a streams-overview list.
Both of that is tackled in this PR.

### Steps for Testing

Prerequisites:
- 1 Students

1. Log in
2. Navigate to a Livestream
3. Open the Playlist Sidebar
4. Check if the current stream is marked as playing
5. Check if the other streams are displayed accordingly
6. Check if the Prev and Next button is working

### Screenshots
Bildschirmfoto 2023-04-19 um 17 51 04

### CODE REVIEW

File: api/stream.go [hunk 1]
```diff
@@ -301,6 +302,32 @@ func (r streamRoutes) getStream(c *gin.Context) {
...
+		CourseSlug string    `json:"courseSlug"`
+		StreamName string    `json:"streamName"`
+		LiveNow    bool      `json:"liveNow"`
+		Watched    bool      `json:"watched"`
+		CreatedAt  time.Time `json:"createdAt"`
+	}
+
+	tumLiveContext := c.MustGet("TUMLiveContext").(tools.TUMLiveContext)
+	var result []StreamPlaylistEntry
+	for _, stream := range tumLiveContext.Course.Streams {
```
Comment (MatthiasReumann) [inline api/stream.go:345]: I think it would make sense to only include VODs in the playlist. Currently livestreams and streams in waiting rooms are displayed too.

File: web/template/watch.gohtml [hunk 2]
```diff
@@ -400,13 +400,22 @@
...
-             :class="showBookmarks ? 'lg:basis-1/4' : 'lg:basis-0'"
+        <div id="bookmarks-desktop" x-cloak="" x-show="sidebar === watch.SidebarState.Bookmarks"
+             :class="sidebar === watch.SidebarState.Bookmarks ? 'lg:basis-1/4' : 'lg:basis-0'"
              class="hidden md:block basis-full h-96 lg:h-16/6 px-5 md:px-2 md:pt-2 lg:order-none order-4">
             {{template "bookmarks-modal" $stream.ID}}
         </div>
         {{end}}
 
+        <!-- Streams -->
+        {{if .IndexData.TUMLiveContext.User}}
```
Comment (MatthiasReumann) [inline web/template/watch.gohtml:413]: I unfortunately can't open playlists on small screens (e.g. mobile), i.e. if I press the playlist button nothing happens.

**Review by MatthiasReumann — DISMISSED** [review]
1001st PR 🚀

**Review by joschahenningsen — APPROVED** [review]
Looks great! You can also include the thumbnails if you want. They are available at /api/stream/{streamID}/thumbs/vod

**Comment by mono424** [comment]
@joschahenningsen @mono424 added thumbnails in new home style :)

File: web/package.json [hunk 3]
```diff
@@ -42,6 +43,7 @@
     "http-status-codes": "^2.2.0",
     "ical.js": "^1.5.0",
     "katex": "^0.16.4",
+    "lint-fix": "^0.24.0",
```
Comment (joschahenningsen) [inline web/package.json]: Why was this added?

File: web/package.json [hunk 4]
```diff
@@ -10,10 +10,11 @@
     "postinstall": "npm run build && npm run tailwind-compile",
     "tailwind-compile": "NODE_ENV=production npx tailwindcss build -i assets/css/main.css --minify -o ./assets/css-dist/main.css && npx tailwindcss build -i assets/css/home.css --minify -o ./assets/css-dist/home.css",
```
Comment (joschahenningsen) [inline web/package.json]:
```suggestion
    "tailwind-compile": "NODE_ENV=production npx tailwindcss build -i assets/css/main.css --minify -o ./assets/css-dist/main.css && npx tailwindcss build -i assets/css/home.css --minify -o ./assets/css-dist/home.css && npx tailwindcss build -i assets/css/watch.css --minify -o ./assets/css-dist/watch.css",
```

**Review by joschahenningsen — CHANGES_REQUESTED** [review]

File: web/package.json (hunk 3 above)
Comment (mono424) [reply web/package.json]: I think I ran `npm i lint-fix` instead of `npm run lint-fix`, sometimes my brain does wired stuff! Thanks for the catch ! 👍

**Review by joschahenningsen — CHANGES_REQUESTED** [review]
I really like how this looks! 
https://1001.test.live.mm.rbg.tum.de/admin/course/4
I noticed that future streams are shown, this is probably undesirable.

Also, I couldn't get the chat to show when enabling the chat for the course. It seems like course.chat_enabled is not set to true, but the chat is now only enabled for every course? This seems unintuitive but unrelated to this pr.

**Review by joschahenningsen — APPROVED** [review]
Like this a lot! Thanks
